charts/hono/values.yaml

```diff
@@ -1217,6 +1217,9 @@ deviceRegistryExample:
     #    secretName: "example-data"
     #    mountPath: "/etc/data"
 
+  # The path to the truststore for clients using this device registry.
```

Comment:
Can you please add some wording that explains under which circumstances this value will be used and for what?